frontend/src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+
+const Button = ({type,children})=>(
+    <button className="btn btn-primary btn-block col-12" type={type}>{children}</button>
```

Comment:
A col-12 layout szintű formázás, lehet nem szerencsés itt megadni, hanem a használat helyén kellene, mert később lehet használnátok más helyen is a gombot, ahol nem 12 oszlop szélesnek kell lennie.